package-lock.json

```diff
@@ -626,7 +626,7 @@
     },
     "node_modules/assert/node_modules/util": {
       "version": "0.10.3",
-      "resolved": "https://registry.npmjs.org/util/-/util-0.10.3.tgz",
+      "resolved": "http://registry.npmjs.org/util/-/util-0.10.3.tgz",
```

Comment:
Is this correct? I'd expect all addresses to be http**_S_**

Reply:
Hum that's a good point. This is automatically generated but Ill ping the `libsbp` channel to see if we can update the generator